src/main/kotlin/blackjack/model/Participant.kt

```diff
@@ -0,0 +1,23 @@
+package blackjack.model
+
+abstract class Participant {
```

Comment:
Is there a specific reason for using an `abstract class` rather than an `interface`?